File: app/src/main/java/com/example/mycake/UserHelperClass.java
package com.example.mycake;

public class UserHelperClass {
  private String fullName, username, password, password2, phoneNumber, email;


  public UserHelperClass(String fullName, String username, String password, String password2, String phoneNumber, String email) {
    this.fullName = fullName;
    this.username = username;
    this.password = password;
    this.password2 = password2;
    this.phoneNumber = phoneNumber;
    this.email = email;
  }

  public UserHelperClass() {
  }

  public String getFullName() {
    return fullName;
  }

  public void setFullName(String fullName) {
    this.fullName = fullName;
  }

  public String getUsername() {
    return username;
  }

  public void setUsername(String username) {
    this.username = username;
  }

  public String getPassword() {
    return password;
  }

  public void setPassword(String password) {
    this.password = password;
  }

  public String getPassword2() {
    return password2;
  }

  public void setPassword2(String password2) {
    this.password2 = password2;
  }

  public String getPhoneNumber() {
    return phoneNumber;
  }

  public void setPhoneNumber(String phoneNumber) {
    this.phoneNumber = phoneNumber;
  }

  public String getEmail() {
    return email;
  }

  public void setEmail(String email) {
    this.email = email;
  }
}

File: app/src/main/java/com/example/mycake/UserHelperClass.kt
package com.example.mycake

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.WindowManager
import com.example.mycake.R
import android.widget.TextView
import android.widget.EditText
import com.google.firebase.database.DatabaseReference
import com.google.firebase.database.FirebaseDatabase
import com.google.firebase.database.ValueEventListener
import com.google.firebase.database.DataSnapshot
import android.content.Intent
import com.google.firebase.database.DatabaseError
import android.widget.Toast
import android.content.DialogInterface
import com.hbb20.CountryCodePicker
import com.google.android.gms.tasks.OnSuccessListener
import com.google.android.gms.tasks.OnFailureListener
import android.annotation.SuppressLint
import android.view.animation.Animation
import android.os.Looper
import android.widget.RelativeLayout

class UserHelperClass {
    var fullName: String? = null
    var username: String? = null
    var password: String? = null
    var password2: String? = null
    var phoneNumber: String? = null
    var email: String? = null

    constructor(
        fullName: String?,
        username: String?,
        password: String?,
        password2: String?,
        phoneNumber: String?,
        email: String?
    ) {
        this.fullName = fullName
        this.username = username
        this.password = password
        this.password2 = password2
        this.phoneNumber = phoneNumber
        this.email = email
    }

    constructor() {}
}